crates/spirv-std/Cargo.toml

```
@@ -12,7 +12,7 @@ spirv-std-types.workspace = true
 spirv-std-macros.workspace = true
 bitflags = "1.2.1"
 num-traits = { version = "0.2.14", default-features = false, features = ["libm"] }
```

```suggestion
num-traits = { version = "0.2.15", default-features = false, features = ["libm"] }
```
... this is all it takes to make `0.23` work, too, I'm so sorry for not noticing this earlier!